docs/networking/best-practice-non-vlan-aware-switch.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 51
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+Description: Harvester is built on Kubernetes, which uses CNI as an interface between network providers and Kubernetes pod networking. Naturally, we implement the Harvester network based on CNI. Moreover, the Harvester UI integrates the Harvester network to provide a user-friendly way to configure networks for VMs.
+---
+
+# Best Practice for Non VLAN-aware Switch
+
+In this best practice guide for non VLAN-aware switch, dummy switch, we will introduce Harvester VLAN network and external switch configuration for common scenario.
```

Comment:
in my previous job (network engineer), we usually said "non vlan aware" or "dummy" rather than "vlan unaware".
i'm not sure if it will confuse network team or not.

about "vlan-unaware" vs "dummy"
i'm asking my ex-colleague about this.
but we could remove "dummy" and leave only "vlan-unaware" (or non vlan aware)

Reply:
I'll trust you here :) ... it's just that something "non ..." always reads weird. But if it's a terminology, please keep it like that.

As for the dummy reference, we can keep it, the phrasing would change to this:
"In this best practice guide for non VLAN-aware switch, also known as dummy switch, ..."

Reply:
> "In this best practice guide for non VLAN-aware switch, also known as dummy switch, ..."

this should be good

Reply:
I edited all the comments with VLAN-unaware and changed it back to "non VLAN-aware". You can review them and commit them if it's OK

Reply:
Sure ok
thanks a lot